client/src/pages/Unsubscribe/Unsubscribe.jsx

```diff
@@ -0,0 +1,65 @@
+import { React, useState, useContext } from 'react';
+import { TextInput } from '../../components/input/TextInput/TextInput';
+import { Button } from '../../components/button/Button/Button';
+import { SnackbarContext } from '../../util/SnackbarProvider';
+import './unsubscribe.scss';
+
+import { useDispatch } from 'react-redux';
+import { unsubscribeUser } from '../../state/user/saga';
+
+const PageUnsubscribe = () => {
+  const [email, setEmail] = useState('');
+  const { setSnackbar } = useContext(SnackbarContext);
+
+  const dispatch = useDispatch();
+
+  const validateEmail = (email) => {
+    return String(email)
+      .toLowerCase()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
can you use the package 'email-validator' here instead like we do on the backend? (best to have consistency there).